glusterlib/playbook_gen.py

```diff
@@ -136,19 +136,15 @@ def move_templates_to_playbooks(self):
         /tmp/playbooks
         '''
         # Is the templates present as a part of ansible installation?
-        templates_path_pkg = '/usr/share/ansible/ansible-glusterfs/templates'
-        # Or is it present in the source directory or installed via setuptools
-        templates_path_bk = self.get_file_dir_path(self.uppath(__file__, 2),
-                                                   'templates')
```

Comment:
Also the templates has a bk directory keeping in mind that the templates can be stored in the current working directory and can be run. For someone who is developing or using this without installing the whole package, this is much easier a way. If I change a playbook, installing it every time to test seems to be a hard way.